wolfcrypt/test/test.c

```diff
@@ -426,6 +426,13 @@ const byte const_byte_array[] = "A+Gd\0\0\0";
 #ifdef DEVKITPRO
     #include <wiiuse/wpad.h>
 #endif
+#ifdef NDS
```

Comment:
Is `NDS` a compiler pre-processor macro that gets defined?

Reply:
It gets manually defined as descibed in ./IDE/NDS/README.md .
I'm sorry if it's not the correct way to do it, but I found it as the best way to seperate the parts for cross compilation.

Reply:
I see. It's not specific enough. Maybe change to WOLFSSL_NDS. Did you see the existing `DEVKITPRO` build option? Curious if you tried with that or maybe there is a way to expand that existing one?

Reply:
I will change that, as for the existing `DEVKITPRO` build option, it seems to be for the Wii and is incompatible. at least for displaying the tests.

Reply:
Shuld i really change it? For what i understand `WOLFSSL_xxx` seems to be for features, and just `xxx` for diffrent plattforms, as for the existing `DEVKITPRO` build option, it seems to be for the Wii and is incompatible. at least for displaying the tests.

Reply:
You can keep it as NDS for now. Thank you